sites/shared/components/workbench/header.mjs

```diff
@@ -59,7 +59,10 @@ const NavIcons = ({ setView, setDense, dense, view }) => {
         extraClasses="text-success bg-neutral hover:bg-success hover:text-neutral"
       >
         {dense ? (
-          <RightIcon className={`${iconSize} animate-bounce-right`} stroke={4} />
+          <RightIcon
+            className={`${iconSize} group-hover:animate-[bounceright_1s_infinite] animate-[bounceright_1s_5]`}
```

Comment:
This is better. But we should probably stick this animation behind `motion-safe:` to be even better